driver/src/main/scala/api/MongoDriver.scala

```diff
@@ -198,6 +210,17 @@ class MongoDriver(
   /**
    * Creates a new MongoConnection from URI.
    *
+   * @param uri the URI to be parsed by [[reactivemongo.api.MongoConnection.parseURI]]
+   * @param name $connectionNameParam
+   */
+  def askConnection(uri: String, name: Option[String]): Future[MongoConnection] = MongoConnection.parseURI(uri) match {
```

Comment:
`uriStrict` ?

Reply:
yeah, uri is now always strict for new askConnection API, so maybe we should simply use `uri` as param name everywhere in the new api? or do you prefer we use `uriStrict`?